core/scripts/PrecisionErrors.js

```diff
@@ -642,20 +662,21 @@ async function runExport() {
   //       therefore the proportion of collateral returned is less than the synthetic burned: `(C - c)/C < (T-S)/T`.
   //   (2) Identically to a withdraw, `c` collateral is sent from the contract to the user,
   //       while `(c / M)` raw collateral is removed from `RC`. `(c / M)` is floored and therefore
-  //       the contract loses more collateral than it debits from sponsors: `(RC - c/M) * M > (C - c)`.
-  // Error Amount:
-  // -1e-17 in `c` if `(S / T)` has -1e-18 error and is then multiplied by C's least significant decimal
-  // +1e-18 in `RC` for the same reason as in a withdraw.
-  // Here's an example of (1):
+  //       the contract loses more collateral than it debits from sponsors: `(RC - c/M) * M > (C - c)`. We intentionally
+  //       deal with this error in the raw collateral the same way we would with a `withdraw`, so we won't demonstrate
+  //       this error in this test.
+  // Example of (1):
   // - M = cumulativeFeeMultiplier = 1
   // - RC = rawCollateral = 9
   // - T = tokensOutstanding = 9
   // - S = amountToRedeem = 8
   // - (S / T) = 0.888...repeating and gets floored to 0.888...88
   // - (S / T) * (RC * M) = 7.999...92, instead of 8
   // - So, the user burns 8 synthetic but receives 7.999...92 collateral, representing 8e-18 less than they were expecting.
+  // Error Amount:
+  // - (-1e-17) in `c` if `(S / T)` has -1e-18 error and is then multiplied by C's least significant decimal
 
-  console.group("Precision loss in proportion of collateral redeemed via redeem()");
+  console.group("Precision loss in % of collateral redeemed via redeem()");
```

Comment:
Great to express it as a % of the total collateral redeemed, as you do in you comments above.